app/services/cclf/fee/simple_fee_adapter.rb

```diff
@@ -0,0 +1,42 @@
+module CCLF
+  module Fee
+    class SimpleFeeAdapter
+      attr_reader :object
+
+      def initialize(object)
+        @object = object
+      end
+
+      def bill_type
+        raise 'Implement in sub-class'
+      end
+
+      def bill_subtype
+        raise 'Implement in sub-class'
+      end
+
+      def bill_scenario
+        case_type_adapter.bill_scenario
+      end
+
+      private
+
+      def case_type_adapter
+        ::CCLF::CaseTypeAdapter.new(object.claim.case_type)
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2221/files#r165341816)